### PR TITLE
Content-Type must be present on writes

### PR DESCRIPTION
**Pull request by kjetilk**
This is my first stab at a test case for the first test case that can be done in the spec. Lets try to make this work, then next is probably easy :-)

### CODE REVIEW

File: protocol/solid-protocol-spec.ttl [hunk 1]
```diff
@@ -10,11 +10,21 @@
...
                    <https://solidproject.org/TR/protocol#web-access-control>
 .
 
+<https://solidproject.org/TR/protocol#sr-http-content-type>
+  a spec:NormativeRequirement ;
+  spec:conformanceRole spec:ServerConformanceClass ;
+  spec:normativeConformanceLevel spec:ConformanceMust ;
+  schema:name "Content-Type on Writes" ;
+  schema:description "Full text of the specification section" ;
+  spec:excerpt "A Solid server MUST reject PUT, POST and PATCH requests without the Content-Type header with a status code of 400."
```
Comment (edwardsph) [inline protocol/solid-protocol-spec.ttl:25]: The spec vocab has changed (https://github.com/solid/vocab/blob/specification-terms/spec.ttl) - I just updated the files in this repo.
```
    spec:requirementSubject spec:Server ;
    spec:requirementLevel spec:MUST ;
    spec:statement "excerpt of requirement 1"@en
```
Can you rebase this PR on the latest commit?

**Review by edwardsph — CHANGES_REQUESTED** [review]
LGTM - approved after rebase and vocab changes

**Comment by kjetilk** [comment]
There is unfortunately a short-coming in Karate here, it adds the `Content-Type` header by default. It is straightforward to set the header to an empty string, but not to remove it entirely. So, this seems to be the best we can do.

File: protocol/solid-protocol-spec.ttl [hunk 2]
```diff
@@ -10,32 +10,42 @@
...
+    spec:requirement <https://solidproject.org/TR/protocol#sr-http-content-type>,
+                   <https://solidproject.org/TR/protocol#resource-representations>,
                    <https://solidproject.org/TR/protocol#writing-resources>,
                    <https://solidproject.org/TR/protocol#web-access-control>
 .
 
+<https://solidproject.org/TR/protocol#sr-http-content-type>
+  a spec:NormativeRequirement ;
+  spec:conformanceRole spec:ServerConformanceClass ;
+  spec:normativeConformanceLevel spec:ConformanceMust ;
```
Comment (edwardsph) [inline protocol/solid-protocol-spec.ttl:22]: These are still old vocab

**Review by edwardsph — APPROVED** [review]
LGTM

**Pull request merged**